data/notices.json

```
@@ -887,6 +887,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(aws-lambda-nodejs): Cannot find defining file",
+      "issueNumber": 34486,
+      "overview": "The addition of the `@propertyInjectable` decorator onto `NodejsFunction` breaks it because it alters the callstack, which is inspected inside the construct to automagically determine the entrypoint of the function.",
```

This should be the issue overview, not the PR (the user doesn't really care about why it broke...)

```suggestion
      "overview": "Instantiation of NodejsFunction fails when the 'entry' property is not passed",
```